### ISSUE TITLE
Post-quantification or Aware-quantization?

### ISSUE THREAD

**Issue by Scotter-Qian** [Zhen-Dong/HAWQ#19]
Excuse me, is this method only applicable to post-quantization?

**Comment by DextroLaev** [comment]
No, first they allocate the bit precision and then they do few epochs of QAT to get back the accuracy.
